aws/rust-runtime/aws-auth/src/provider.rs

```diff
@@ -16,18 +27,30 @@ use std::fmt::{Debug, Display, Formatter};
 use std::future::{self, Future};
 use std::pin::Pin;
 use std::sync::Arc;
+use std::time::Duration;
+
+// Re-export AsyncSleep and TokioSleep so that an explicit dependency on smithy-http isn't needed
+pub use smithy_http::sleep::AsyncSleep;
```

Comment:
Worth noting, this adds a public API dependency on `smithy-http`, so breaking change releases will need to be coordinated.